blocks/init/src/Blocks/components/accordion/assets/accordion.js

```diff
@@ -12,7 +12,6 @@ export class Accordion {
 
 		this.ATTR_HIDDEN = 'aria-hidden';
 		this.ATTR_OPEN = 'data-accordion-open';
-		this.ATTR_PREVENT_CLOSE = 'data-accordion-prevent-close';
```

Comment:
Sure this is unsued?

Reply:
yup, couldn't find an instance of this anywhere 🤷🏼‍♂️